controllers/grid/users/author/AuthorGridHandler.inc.php

```diff
@@ -234,10 +234,10 @@ public function setDataElementSequence($request, $rowId, $gridDataElement, $newS
         if (!$this->canAdminister($request->getUser())) {
             return;
         }
-        $authorDao = DAORegistry::getDAO('AuthorDAO'); /** @var AuthorDAO $authorDao */
-        $author = $authorDao->getById($rowId);
+
+        $author = Repo::author()->get($rowId);
         $author->setSequence($newSequence);
-        $authorDao->updateObject($author);
+        Repo::author()->edit($author, []);
```

Comment:
In this case, I'd prefer the edit to look like this:

```php
Repo::author()->edit($author, ['seq' => $newSequence]);
```

That will ensure the changed prop is passed correctly to the hook.

Reply:
Fixed